ldk/javascript/examples/self-test-loop/src/tests/filesystem/index.ts

```diff
@@ -293,3 +293,54 @@ export const testFileExists = (): Promise<boolean> =>
         });
     });
   });
+
+export const testOpenFile = (): Promise<boolean> =>
+  new Promise(async (resolve, reject) => {
+    const filePath = `test.txt`;
+    const writeMode = 0o755;
+
+    try {
+      await filesystem.writeFile({
+        path: filePath,
+        data: 'some text',
+        writeOperation: filesystem.WriteOperation.overwrite,
+        writeMode,
+      });
+      await filesystem.openWithDefaultApplication(filePath);
+      sleep(2000);
```

Comment:
why not use setTimeout() since we already have this concept in the selfTestLoop?